.murdock

```diff
@@ -76,21 +76,38 @@ get_supported_boards() {
     done | $(_greplist $BOARDS)
 }
 
-# given an app dir as parameter, print "$appdir board" for each supported
-# board. Only print for boards in $BOARDS.
-get_app_board_pairs() {
+get_supported_toolchains() {
+    local board=$1
+
+    # TODO provide make target in similar fashion as the info-boards-supported one
+    for b in native samr21-xpro nucleo-f401re
```

Comment:
I would prefer to have this list in a global variable outside of the function to not have this "magic" list in the function.

Reply:
+1, similar to TEST_BOARDS_AVAILABLE

Reply:
Ok